### PR TITLE
clarify

### PR DESCRIPTION
**Pull request by guillermooo**
Things that I found unclear:

* missing info where routes come from
* *that will be used for request processing* (the URL or the values?)
* URL vs link (better to use just one term?)

My version is longer. I tried to unify terminology and emphasize that Routing can do one op and its reverse.

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[aspnetcore/fundamentals/routing.md](https://github.com/guillermooo/Docs/blob/patch-4/aspnetcore/fundamentals/routing.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C12%5C6%5C6a0d29ae-bb37-09f1-c55a-4cfa862c6784%5CPullRequest%5C201612061031271858-2325%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: aspnetcore/fundamentals/routing.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ uid: fundamentals/routing
 
 By [Ryan Nowak](https://github.com/rynowak), [Steve Smith](http://ardalis.com), and [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-Routing is used to map requests to route handlers. Routes are configured when the application starts up, and can extract values from the URL that will be used for request processing. Routing functionality is also responsible for generating links using the defined routes in ASP.NET apps.
+Routing functionality is responsible for mapping an incoming request to a route handler. Routes are defined in the ASP.NET app and configured when the app starts up. A route can optionally extract values from the URL contained in the request, and these values can then be used for request processing. Using route information from the ASP.NET app, the routing functionality is also able to generate URLs that map to route handlers. Therefore, routing can find a route handler based on a URL, or the URL correspoding to a given route handler based on route handler information.
```
Comment (ardalis) [inline aspnetcore/fundamentals/routing.md]: corresponding

**Comment by guillermooo** [comment]
@ardalis fixed

**Comment by Rick-Anderson** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[aspnetcore/fundamentals/routing.md](https://github.com/guillermooo/Docs/blob/patch-4/aspnetcore/fundamentals/routing.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C12%5C6%5C6a0d29ae-bb37-09f1-c55a-4cfa862c6784%5CPullRequest%5C201612061421125276-2325%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by Rick-Anderson** [comment]
@guillermooo  great work, THANKS!

**Pull request merged**